single.php

```diff
@@ -58,36 +57,6 @@
 	get_home_url()
 );
 
-// Build the shortcode for articles block.
-if ( 'yes' === $post->include_articles ) {
-	$tag_id_array = [];
-	foreach ( $post->tags() as $post_tag ) {
-		$tag_id_array[] = $post_tag->id;
-	}
-	$category_id_array = [];
-	foreach ( $post->terms( 'category' ) as $category ) {
-		$category_id_array[] = $category->id;
-	}
-
-	$block_attributes = [
-		'exclude_post_id' => $post->ID,
-		'tags'            => $tag_id_array,
-		'post_categories' => $category_id_array,
```

Comment:
In old implementation we pass the tags and categories as a block attributes, but as we append the block code at the end of post content, its good that we fetch the post tags and cat. run time within articles block. The articles block changes are [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/795/files#diff-f93c27594f759a20cd2ce6b650aac86492c4a7f8c05c0184d7bfb00c0f385413R321-R327).